src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -28,6 +28,15 @@ public TenantCollection get(Integer offset, Integer limit) {
     return result;
   }
 
+  @Transactional
```

Comment:
We just saving 1 entity to DB, we can do it without transaction

Reply:
fixed